### PR TITLE
we didn't make this mobile friendly for super super small phones

### PR DESCRIPTION
**Pull request by MandyMeindersma**
our tabs were all over the place, some are 4 some are 2. So I just made it all 2

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -16,36 +17,38 @@
...
   </div>
   <main>
     <p>YEGWIT aims to build a strong and safe group for women and allies in tech
-      with the goal of fostering engagement, providing mentorship, and
-      creating community. We want to establish a group that allows individuals
-      to share ideas, concerns, thoughts and actions, to create an
-      equitable environment within the industry.</p>
-    <p>YEGWIT: Foster, Validate, Empower.</p>
+        with the goal of fostering engagement, providing mentorship, and
+        creating community. We want to establish a group that allows individuals
```
Comment (nathany) [inline layouts/index.html]: Instead of "we want to", I'd say "We are establishing..." or something like that.

**Review by nathany — APPROVED** [review]
Thanks Mandy. You rock!

**Comment by MandyMeindersma** [comment]
@nathany added that a couple more word changes from todays meeting

**Comment by nathany** [comment]
:shipit:

**Pull request merged**